src/test/java/com/keniu/BaseIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package com.keniu;
+
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+
+public abstract class BaseIntegrationTest {
+
+    private static final String DB_IMAGE = "mysql:8";
+    private static final MySQLContainer<?> MYSQL_CONTAINER;
```

Comment:
It's not a constant, because we can set some properties and change it
```suggestion
    private static final MySQLContainer<?> mySqlContainer;
```

Reply:
Can be both but ok